server/plugin/command.go

```diff
@@ -267,25 +267,36 @@ func (p *Plugin) handleSubscriptionsList(_ *plugin.Context, args *model.CommandA
 		}
 		txt += "\n"
 	}
-
+	subscription, err := p.GetNotificationTurnedOffRepo()
+	if err != nil {
+		return err.Error()
+	}
+	for _, repo := range subscription {
+		txt += fmt.Sprintf("* `%s` - %s", strings.Trim(repo, "/"), "notification : disabled")
+		txt += "\n"
+	}
 	return txt
 }
 
 func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs, parameters []string, userInfo *GitHubUserInfo) string {
 	features := "pulls,issues,creates,deletes"
 	flags := SubscriptionFlags{}
-
+	var isExcludeRepo bool
+	var excludeRepo string
 	if len(parameters) > 1 {
 		var optionList []string
-
 		for _, element := range parameters[1:] {
-			if isFlag(element) {
+			switch {
+			case parseFlag(element) == "exclude":
+				isExcludeRepo = true
```

Comment:
Could this make use of `SubscriptionFlags`?

Reply:
Sure. , have made the changes